rules/execute-command.js

```diff
@@ -95,13 +95,13 @@ function runCommand(command, args = []) {
     if (command.includes('${NODE}')) {
       command = command
         .split('${NODE}')
-        .join(`node -r ${join(rootDir, '.pnp.js')}`);
+        .join(`${node} -r ${join(rootDir, '.pnp.js')}`);
     }
     if (command.includes('${ROOT_DIR}')) {
       command = command.split('${ROOT_DIR}').join(rootDir);
     }
     try {
-      exec(command, options);
+      exec(`if [[ -x $(which "${command}") ]]; then "${command}"; fi`, options);
```

Comment:
This kind of logic will add additional overhead. Could we instead just say that invalid build commands are not supported?

Reply:
I'm fine with removing this side of the branch altogether, but I thought this logic was here to fall back to executing bash commands if a corresponding package.json script was not found. Do we not need this logic anymore?

Reply:
We still need this branch, but I don't think we need to add this custom logic. If our build is set up in a way which running `command` fails, the build should just fail.